en/04_Changelogs/5.0.0.md

```diff
@@ -781,9 +781,9 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink)
 - Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink)
 - Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink)
-- Removed deprecated class `SilverStripe\CMS\Controllers\CMSPageHistoryController` - use `silverstripe/versioned-admin` instead
-- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileExtension` - use `FileLinkTracking` instead
-- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileFormFactoryExtension` - use `UsedOnTable` instead
+- Removed deprecated class `SilverStripe\CMS\Controllers\CMSPageHistoryController` - use [`SilverStripe\VersionedAdmin\Controllers\CMSPageHistoryViewerController`](api:SilverStripe\VersionedAdmin\Controllers\CMSPageHistoryViewerController) instead
+- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileExtension` - use [`SilverStripe\Assets\Shortcodes\FileLinkTracking`](api:SilverStripe\Assets\Shortcodes\FileLinkTracking) instead
+- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileFormFactoryExtension` - use `UsedOnTable` instead Extension applied to [`SilverStripe\CMS\Model\FileFormFactory`](api:SilverStripe\CMS\Model\FileFormFactory) to decorate with a "Used on:" information area. Uses tracking provided by [`SilverStripe\CMS\Model\SiteTreeFileExtension`](api:SilverStripe\CMS\Model\SiteTreeFileExtension) to generate this
```

Comment:
This is way too long and doesn't fit the established pattern all the other wording uses.
```suggestion
- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileFormFactoryExtension` - use [`SilverStripe\Admin\Forms\UsedOnTable`](api:SilverStripe\Admin\Forms\UsedOnTable) instead
```
If people need more information about what this is and how it's working, they can look at the source code. This points them in the right direction, which is all it needs to do.

Reply:
Done